api/app/services/ingestions/context.rb

```diff
@@ -0,0 +1,32 @@
+module Ingestions
+  class Context
+    include Ingestions::Concerns::Loggable
+    include Ingestions::Concerns::FileOperations
+
+    attr_reader :ingestion, :logger, :source_path, :identifier, :user
+
+    def initialize(ingestion, logger = Rails.logger)
+      @ingestion = ingestion
+      @logger = logger
+      @user = current_user
+
+      initialize_file_operations
+    end
+
+    def initialize_file_operations
```

Comment:
Not loving this method name. It's not really initializing "file operations" so much as staging files in the source dir.